### PR TITLE
feat(doc) : Improvements in the README in order to address the comments made in the merged PR 66

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://github.com/aerogear/mobile-security-service/pull/66

## What
Improvements in the README in order to address the comments made in the merged PR 66

## Verification Steps
Check the changes in this PR in the view mode. 
Please, feel free to review all README. We can address other improvements as well. 

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [X] Finished task
- [] TODO

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -209,6 +210,11 @@ https://www.lucidchart.com/documents/edit/6293d791-8f70-47bb-8136-438128c27129/0
 
 === Using Swagger UI
 
+==== By browser
+
+The swagger api doc is generated in link:./api/swagger.yaml[/api/swagger.yaml] and you can check the REST API definition with this file by use the https://petstore.swagger.io/[Demo] tool online for swaggerUI or https://chrome.google.com/webstore/detail/swagger-ui-console/ljlmonadebogfjabhkppkoohjkjclfai?hl=en[Chrome extension].
```
Comment (laurafitzgerald) [inline README.adoc]:
```suggestion
by using the
```

**Comment by laurafitzgerald** [comment]
Hi @camilamacedo86 I'm leaving some comments here bu I'm also reviewing the linked pr for any other feedback items.

**Comment by laurafitzgerald** [comment]
@camilamacedo86 I can't link it here but there's a line at the bottom of the Pre-requisite section about the contributing guide which should be removed. I thought we did it before but it's duplicate of one at the bottom and I think it's misleading as the user is navigated away from the README after doing pre-reqs which they shouldn't be, they should continue on with the steps below.

File: README.adoc [hunk 2]
```diff
@@ -37,6 +37,7 @@ This is the server component of the AeroGear Mobile Security Service. It is a RE
 |https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
 |https://docs.docker.com/compose/install/[Install the dep package manager]
 |https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
+|https://www.postgresql.org/[PostgreSQL] database
```
Comment (laurafitzgerald) [inline README.adoc]: This pre-req is the only one that doesn't start with a verb, what does the user need to do with Postgres?

Comment (camilamacedo86) [reply README.adoc]: @laurafitzgerald without the Postgres is not possible to start the server.

Comment (laurafitzgerald) [reply README.adoc]: @camilamacedo86 Agreed but the question is more around the way it's asked compared to the others.
You say 
`Install Docker and Docker Compose`
and 
`Install the dep package manager`
and then 
`Postgres`
Should the user install it or run it maybe?

File: README.adoc [hunk 3]
```diff
@@ -37,6 +37,7 @@ This is the server component of the AeroGear Mobile Security Service. It is a RE
 |https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
 |https://docs.docker.com/compose/install/[Install the dep package manager]
```
Comment (laurafitzgerald) [inline README.adoc]: Is this the correct link? Seems to be the same as the one below

File: README.adoc (hunk 2 above)
Comment (finp) [reply README.adoc]: maybe it isn't a pre-req, given that the install process is discussed below?

**Comment by camilamacedo86** [comment]
@laurafitzgerald I did the changes. Please, could you check if all is ok now or if it has anything else that you think that should be changed here?

File: README.adoc (hunk 2 above)
Comment (laurafitzgerald) [reply README.adoc]: @finp Yes I agree that we should have it as _either_ a pre-req or list the install process.

File: README.adoc (hunk 1 above)
Comment (laurafitzgerald) [reply README.adoc]: @camilamacedo86 This conversation is resolved but it looks like the change is still the same?

**Comment by laurafitzgerald** [comment]
@camilamacedo86 Thanks for the changes. I left a comment that one of them may not be resolved still. I left some comments in the other pr. Do you plan to resolve those here also?

File: README.adoc [hunk 4]
```diff
@@ -411,10 +422,6 @@ image::https://user-images.githubusercontent.com/7708031/53174794-8878a600-35e2-
 * https://golang.org/[Golang] - Programming language used
 * https://echo.labstack.com/[Echo] - Web framework used
 
-== Contributing
```
Comment (laurafitzgerald) [inline README.adoc]: @camilamacedo86 I think we should keep this section in and remove the line about contributing from this section: https://github.com/aerogear/mobile-security-service#prerequisites

File: README.adoc [hunk 5]
```diff
@@ -86,26 +86,24 @@ $ docker-compose up -d db
 ----
 
 NOTE: When the database is started the link:./pkg/db/db.go[db.go] routine will be executed and the database model will be created.
-TIP: You can use the https://www.pgadmin.org/[pgadmin] to work with the database. See link:#installing-and-configuring-client-tool-for-the-database[Installing and configuring client tool for the database] how to setup it in your development environment.
+TIP: TIP: You can use https://www.pgadmin.org/[pgadmin] to work with the database. See link:#installing-and-configuring-client-tool-for-the-database[Installing and configuring client tool for the database]
```
Comment (laurafitzgerald) [inline README.adoc]: Two `TIP`'s here

File: README.adoc (hunk 4 above)
Comment (camilamacedo86) [reply README.adoc]: 👍 catcher . Done.

File: README.adoc (hunk 5 above)
Comment (camilamacedo86) [reply README.adoc]: good eyes

Comment (camilamacedo86) [reply README.adoc]: 👍 done

File: README.adoc (hunk 1 above)
Comment (camilamacedo86) [reply README.adoc]: Sorry, you can check now.

**Comment by laurafitzgerald** [comment]
@camilamacedo86 Almost there! I resolved the conversations in the previous pr to mark the ones that have been address. There is one only outstanding which I pinged you on.

**Comment by camilamacedo86** [comment]
@laurafitzgerald the last detail was fixed. Please, let me know if you find something more here.

**Comment by laurafitzgerald** [comment]
@camilamacedo86 thanks for the fix. I've resolved the conversation in the previous pr. I also noticed that some of the tables have an extra title on them which duplicate headings above them. Pre-requisites is one of them. Could we remove the extra titles? e.g. https://github.com/aerogear/mobile-security-service/blob/3179066e1b086e846f87da23552bd755c6df88a8/README.adoc#prerequisites

**Comment by camilamacedo86** [comment]
Hi @laurafitzgerald ,

The titles from the tables were removed too. Please, let me know if you agree in move forward know.

File: README.adoc [hunk 6]
```diff
@@ -31,16 +31,13 @@ This is the server component of the AeroGear Mobile Security Service. It is a RE
...
-|https://docs.docker.com/compose/install/[Install the dep package manager]
+|https://golang.github.io/dep/docs/installation.html/[Install the dep package manager]
 |https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
 |===
 
-See the https://aerogear.org/community/#guides[Contributing Guide] for more information.
-
 == Getting Started
 
 If you'd like to simply run the entire application in `docker-compose`, follow link:#Run-Entire-Application-with-Docker-Compose[these instructions].
```
Comment (laurafitzgerald) [inline README.adoc]: @camilamacedo86 apologies for pinging on changes that weren't part of your pr but this link here is not working correctly. Would you mind updating it here?

File: README.adoc [hunk 7]
```diff
@@ -209,6 +206,11 @@ https://www.lucidchart.com/documents/edit/6293d791-8f70-47bb-8136-438128c27129/0
```
Comment (laurafitzgerald) [inline README.adoc]: The lucid chart link is only available internally. Should we remove it as the image shows what's in the chart anyway?

File: README.adoc (hunk 6 above)
Comment (camilamacedo86) [reply README.adoc]: Hi @laurafitzgerald I tested it now and it is working perfectly. 
See here: https://github.com/aerogear/mobile-security-service/blob/1d30f18d17b6525979826628045290a8ca3870cf/README.adoc

File: README.adoc (hunk 7 above)
Comment (camilamacedo86) [reply README.adoc]: 👍 Done

File: README.adoc (hunk 6 above)
Comment (laurafitzgerald) [reply README.adoc]: @camilamacedo86 I'm sure why but when I click on the `these instructions` link the url updates but it doesn't navigate to the section.

Comment (camilamacedo86) [reply README.adoc]: screenshot 2019-02-25 at 10 47 08

Comment (camilamacedo86) [reply README.adoc]: Fixed 👍

Comment (laurafitzgerald) [reply README.adoc]: Confirmed the link works correctly now. Thanks.

**Pull request merged**